pcap4j-core/src/main/java/org/pcap4j/packet/Dot11ProbeRequestPacket.java

```diff
@@ -474,20 +475,23 @@ public Dot11ProbeRequestPacket build() {
      */
     private static final long serialVersionUID = -2203820242563461514L;
 
-    private final Dot11SsidElement ssid;
-    private final Dot11SupportedRatesElement supportedRates;
-    private final Dot11RequestElement request;
-    private final Dot11ExtendedSupportedRatesElement extendedSupportedRates;
-    private final Dot11DsssParameterSetElement dsssParameterSet;
-    private final Dot11SupportedOperatingClassesElement supportedOperatingClasses;
-    private final Dot11HTCapabilitiesElement htCapabilities;
-    private final Dot112040BssCoexistenceElement twentyFortyBssCoexistence;
-    private final Dot11ExtendedCapabilitiesElement extendedCapabilities;
-    private final Dot11SsidListElement ssidList;
-    private final Dot11ChannelUsageElement channelUsage;
-    private final Dot11InterworkingElement interworking;
-    private final Dot11MeshIdElement meshId;
-    private final List<Dot11VendorSpecificElement> vendorSpecificElements;
+    private Dot11SsidElement ssid = null;
+    private Dot11SupportedRatesElement supportedRates = null;
+    private Dot11RequestElement request = null;
+    private Dot11ExtendedSupportedRatesElement extendedSupportedRates = null;
+    private Dot11DsssParameterSetElement dsssParameterSet = null;
+    private Dot11SupportedOperatingClassesElement supportedOperatingClasses = null;
+    private Dot11HTCapabilitiesElement htCapabilities = null;
+    private Dot112040BssCoexistenceElement twentyFortyBssCoexistence = null;
+    private Dot11ExtendedCapabilitiesElement extendedCapabilities = null;
+    private Dot11SsidListElement ssidList = null;
+    private Dot11ChannelUsageElement channelUsage = null;
+    private Dot11InterworkingElement interworking = null;
+    private Dot11MeshIdElement meshId = null;
+    private List<Dot11VendorSpecificElement> vendorSpecificElements = new ArrayList<Dot11VendorSpecificElement>();
+
+    private List<Dot11InformationElement> infoElementOrderedList = new LinkedList<Dot11InformationElement>();
```

Comment:
infoElementOrderedList holds the same objects as the other fields such as meshId, which is a bad practice.